relayer/chain/ethereum/config.go

```diff
@@ -16,3 +18,8 @@ type ChannelConfig struct {
 	Inbound  string `mapstructure:"inbound"`
 	Outbound string `mapstructure:"outbound"`
 }
+
+type ContractsConfig struct {
+	PolkadotRelayChainBridge string `mapstructure:"polkadot_relay_chain_bridge"`
+	ValidatorRegistry        string `mapstructure:"validator_registry"`
```

Comment:
is this really even used?

Reply:
Not used for now. The production version could/should check that the validator address passes `ValidatorRegistry`'s validation before making the call initial verification call to `PolkadotRelayChainBridge`. But can remove it from the config for now until we bring validation back into the contracts.

Reply:
yeh - tbh i dont even know if we need this for production version

Reply:
Removed in https://github.com/Snowfork/polkadot-ethereum/pull/314/commits/e380f1392bab16b837d7bb5da653103eede3b5b2